functions/streamtoidv/function.dev.json

```diff
@@ -3,8 +3,13 @@
   "handler": "main.handle",
   "description": "The change integration stream consumer that updates the identity vault",
   "role": "arn:aws:iam::656532927350:role/CISLambdaFunctionRoles-StreamToIDV-T5IH2261NJZM",
+  "environment": {
+    "CIS_ARN_MASTER_KEY": "arn:aws:kms:us-west-2:656532927350:key/9e231aa0-04e4-4517-a45d-633c3bb055f0",
+    "CIS_DYNAMODB_TABLE": "CISStreamandDB-IdentityVaultUsers-AQ358T5XZNSQ",
+    "CIS_KINESIS_STREAM_ARN": "arn:aws:kinesis:us-west-2:656532927350:stream/CISStreamandDB-CISInputStream-1FTGC3KE4NN9U"
+  },
   "hooks":{
     "build": "pip3 install -r requirements.txt -t .",
-    "clean": "find . -d -not -name \"..\" -not -name \".\" -not -name \"*.json\" -not -name \"main.py\" -not -name \"function.*\" -not -name \"requirements.txt\" | xargs rm -fr &> /dev/null && find . -type f -name \"*.py\" -name \"*.so\" -not -name \"*.json\" -not -name \"main.py\" -not -name \"function.*\" -not -name \"requirements.txt\" | xargs rm -fr &> /dev/null"
+    "clean": "find . -d -not -name \"..\" -not -name \".\" -not -name \"*.json\" -not -name \"main.py\" -not -name \"function.*\" -not -name \"utils.py\" -not -name \"requirements.txt\" | xargs rm -fr &> /dev/null && find . -type f -name \"*.py\" -name \"*.so\" -not -name \"*.json\" -not -name \"main.py\" -not -name \"utils.py\" -not -name \"function.*\" -not -name \"requirements.txt\" | xargs rm -fr &> /dev/null && find . -type f -name \"*.py\" -name \"*.so\" -not -name \"*.json\" -not -name \"utils.py\" -not -name \"function.*\" -not -name \"requirements.txt\" | xargs rm -fr &> /dev/null"
```

Comment:
@gdestuynder likely... very likely.  You only get a bash one liner here.  I could actually put a script in the directory that just runs each statement so it's more readable.  Step in the right direction?  

Likely should not block merge though.